Automattic/wp-calypso
# Jetpack Connection: Update the old Pressable logo with the new one.

This is a follow-up PR that updates the Jetpack + Pressable connection flow, which was previously missed when we updated the Pressable logo.

| Before | After |
|--------|--------|
| <img width="455" alt="Screenshot 2024-08-07 at 3 49 14 PM" src="https://github.com/user-attachments/assets/f47d0999-2639-4081-ad7a-934167369573"> | <img width="479" alt="Screenshot 2024-08-07 at 3 49 04 PM" src="https://github.com/user-attachments/assets/9ca2f4fa-8d07-4b84-8e4b-f4518dfb585b"> | 



Follow up https://github.com/Automattic/wp-calypso/pull/92938

## Proposed Changes

* Update the Jetpack Header's Pressable component to use the new graphics.

## Why are these changes being made?

* This is to ensure that we present the Pressable brand logo consistently with the updated design.

## Testing Instructions

* Spin up a Pressable site and attempt to connect Jetpack to your WPCOM account. (Make sure you don't proceed with the connection when you see the screen above.
* replace the URL `https://wordpress.com` domain with the Calypso live link below. (e.g. `{CALYPSO_LIVE_LINK}/jetpack/connect/authorize?client_id=235793677&redirect_uri=https%3A%2F%2....`
* Confirm that the Pressable logo has been updated.

## Pre-merge Checklist
